src/entries/popup/pages/home/NFTs/NFTDetails.tsx

```diff
@@ -326,6 +349,78 @@ const NFTPriceSection = ({ nft }: { nft?: UniqueAsset | null }) => {
   );
 };
 
+const NFTAccordionSectionEnsProfile = ({
+  cover,
+  bio,
+  address,
+  twitter,
+  website,
+}: {
+  cover?: string;
+  bio?: string;
+  address?: string;
+  twitter?: string;
+  website?: string;
+}) => {
+  const visitTwitter = () => {
+    goToNewTab({
+      url: `https://www.twitter.com/${twitter}`,
+    });
+  };
+  const visitWebsite = () => {
+    const fragment = website
+      ?.replace('https://', '')
+      .replace('http://', '')
+      .replace('www.', '');
+    goToNewTab({
+      url: `https://www.${fragment}`,
+    });
+  };
+  return (
+    <>
+      <AccordionItem value="profile">
+        <AccordionTrigger>
+          {i18n.t('nfts.details.ens_profile_info')}
+        </AccordionTrigger>
+        <AccordionContent gap="24px">
+          <div />
+          {cover && <ENSProfileInfoCover src={cover} />}
+          {bio && <ENSProfileInfoBio bio={bio} />}
+          {address && (
+            <NFTInfoRow
+              symbol={'person.crop.rectangle.fill'}
+              label={i18n.t('nfts.details.ens_address')}
+              value={truncateAddress(address as AddressOrEth)}
+              valueSymbol={'doc.on.doc'}
+              onClick={() => copyAddress(address as Address)}
+            />
+          )}
+          {twitter && (
+            <NFTInfoRow
+              symbol={'person.crop.rectangle.fill'}
+              label={'Twitter'}
```

Comment:
shouldn't this be `X` now? same for the birdIcon

Reply:
never

Reply:
jk, idk. I thought the same, but we call it Twitter in settings still, so I figured I'd stick with the design for now.